core/src/apps/cardano/address.py

```diff
@@ -114,24 +131,17 @@ def derive_enterprise_address(keychain: seed.Keychain, path: list, network_id: i
     )
     address = address_header + spending_part
 
-    return get_human_readable_address(address)
+    return address
 
 
-def derive_bootstrap_address(keychain: seed.Keychain, path: list) -> str:
+def _derive_bootstrap_address(keychain: seed.Keychain, path: list) -> str:
     if not _validate_bootstrap_address_path(path):
```

Comment:
I am not sure the term `bootstrap address` is used anymore (apart from that rather old design document). The CDDL now uses `byron address`, which I find much clearer.  Let's ask them and if they confirm, we rename it everywhere in the code.